cc/integration/awskms/aws_kms_client.cc

```diff
@@ -90,7 +90,7 @@ StatusOr<std::string> Read(const std::string& filename) {
 // Extracts a value of 'name' from 'line', where 'line' must be in format:
 // name = some_value
 StatusOr<std::string> GetValue(absl::string_view name, absl::string_view line) {
-  std::vector<std::string> parts = absl::StrSplit(line, '=');
+  std::vector<std::string> parts = absl::StrSplit(line, " = ");
```

Comment:
Why was this change needed? Seems it already strips whitespace below

Reply:
`aws_session_token` can sometimes contain "=", and therefore, when splitting the string by "=", we can end up with more than two chunks. Splitting by " = " ensures that we always have two chunks per line.